content/pages/constitution.md

```diff
@@ -0,0 +1,312 @@
+Author: Fatema Boxwala
```

Comment:
Need to have a title metadatum or else this won't build:

```
ERROR: Skipping pages/constitution.md: could not find information about 'NameError: title'
```

Also please add a `hidden: true` keypair. You can actually ditch the summary as it won't display anywhere for pages!